contexts/gs1-8110-coupon/v1.json

```diff
@@ -20,6 +20,7 @@
         "gtin": "https://gs1.org/voc/gtin",
 
         "description": "https://gs1.org/voc/offerDescription",
+        "exclusionDescription": "https://gs1.org/voc/exclusionDescription",
```

Comment:
Modifications to context after a release are prohibited. Any new terms should be included in a new version.

Reply:
Sorry...thought I got those all cleaned up. Will fix.